### ISSUE TITLE
две дисциплины в одной

### ISSUE THREAD

**Issue by pixelmagenta** [ev-br/gia#6]
"Случайные процессы и теория массового обслуживания" преподавались как две отдельных дисциплины: "Теория случайных процессов" в общем цикле и "Теория массового обслуживания" только у групп специализирующихся на стохастических процессах.
Может быть стоит попросить Юлию Борисовну пересмотреть этот раздел?

**Comment by ev-br** [comment]
Да, это правильное замечание. Ю.Б. этот раздел смотрит. Вопрос к вам: верно ли, что указанные в текущей версии вопросы освещались в общей дисциплине ТСП? Если какие-то не освещались, то какие

**Comment by pixelmagenta** [comment]
Все вопросы за исключением последнего освещались на лекциях и семинарах.
Теоретическая часть последнего вопроса была на лекциях, но на семинарах со Шнурковым П.В. мы его не разбирали.
